src/routes/ethereum-navigator/[slug]/+page.svelte

```
@@ -0,0 +1,112 @@
+                            src="https://beehiiv-images-production.s3.amazonaws.com/uploads/user/profile_picture/5b0c5e78-b7a5-4966-afb6-572c8f3304c8/2077_.png"
+                            alt=""
+                            class="w-6 h-6 rounded-full" />
+                        <span class="text-sm text-zinc-400">{author}</span>
+                    </div>
+                {/each}
+            {/if}
+        </div>
+
+        <div class="prose prose-invert prose-lg max-w-none">
+            {#if post.content}
+                {@html post.content}
```

ESLint

[error] `{@html}` can lead to XSS attack.

(svelte/no-at-html-tags)